app/components/placement/summary_component.html.erb

```diff
@@ -1,7 +1,7 @@
 <li class="app-search-results__item">
   <h2 class="app-search-result__item-title">
     <!-- TODO: link to placement details -->
-    <%= govuk_link_to("", no_visited_state: true) do %>
+    <%= govuk_link_to(placements_provider_placement_path(@provider, placement), no_visited_state: true) do %>
```

Comment:
The fact that we have to pass a `@provider` through, just to make this link work... 🤔 it's making me wonder if our routing is suboptimal.

Technically these placements don't "belong to" any specific provider. I wonder if we should be using top-level routes at this point, rather than nesting them inside `/providers/{a-specific-provider}/placements`. The placements themselves don't change based on the context of which provider we're logged in as.

What do you think?

Either way I think it's out of scope for this PR.

Reply:
As we've discussed, moving this routing to a top-level might be premature at this stage, as we don't know the further extend to how this page will be used.
Currently as a provider is the only user who can see this component, it might be worth revisiting in the future.